templates/python/custom-copilot-basic/src/bot.py.tpl

```diff
@@ -43,7 +43,7 @@ planner = ActionPlanner(
 
 # Define storage and application
 storage = MemoryStorage()
-app = Application[TurnState](
+botapp = Application[TurnState](
```

Comment:
minor: maybe `bot_app` to follow Python's naming convention?

Reply:
nice suggestion